_includes/more-information-tiles.html

```diff
@@ -0,0 +1,105 @@
+{%- endunless %}
+{%- assign country_pages = site.pages | where_exp: "item", "item.search_exclude != true" | where_exp:"item","item.national_resources != nil" %}
+{%- unless country_pages.size == 0 %}
+{%- assign tool_matches_total = 0 %}
+{%- assign query = "related_pages." | append: page.type %}
+{%- for country_page in country_pages %}
+{%- assign tool_matches = country_page.national_resources | where_exp:"resource","resource.related_pages != nil" | where: query, page.page_id %}
+{%- assign tool_matches_total = tool_matches_total | plus: tool_matches.size %}
+{%- endfor %}
+{%- endunless %}
+{%- unless tool_matches_total == 0 %}
+{%- assign actual_nat_tools = 1 %}
+{%- endunless %}
+{%- endif %}
+{%- if actual_fairsharing or actual_faircookbook or actual_dsw or actual_rdmkit or actual_dsw or actual_training or actual_tools or actual_nat_tools %}
+<h2 class="mb-4">{{site.theme_variables.headings.more-information-tiles | default: 'More information' }}</h2>
+{%- endif %}
+
+{%- if actual_training %}
+<!-- Training -->
+<a data-bs-toggle="collapse" href="#training_collapse" role="button" aria-expanded="false" aria-controls="training_collapse" class="d-flex align-items-baseline pb-3 border-top info-collapse">
+    <h3 class="mb-0 flex-grow-1 mt-3 text-dark">Training</h3><i class="fa-solid fa-chevron-down fs-5"></i>
+</a>
+<div class="collapse info-card" id="training_collapse">
+    <div class="mt-2 p-3 pb-5">
+
+        <div class="row row-cols-1 row-cols-md-2 gy-2 gx-4 info-links">
+            {%- for training in page.training %}
+            <div class="col">
+                <a class="btn d-grid h-100" href="{{ training.url }}">
+                    <div class="row g-1">
+                        <div class="col-2 d-flex align-items-center justify-content-center">
+                            {%- assign registry = training.registry | downcase %}
+                            {%- if registry == "tess" %}
+                            <img alt="TeSS logo" class="img-fluid" src="{{ 'assets/img/tess_logo.svg' | relative_url }}">
+                            {%- elsif registry == "zenodo" %}
+                            <img alt="Zenodo logo" class="img-fluid" src="{{ 'assets/img/zenodo_logo.svg' | relative_url }}">
+                            {%- elsif registry == "youtube" %}
+                            <img alt="YouTube logo" class="img-fluid" src="{{ 'assets/img/youtube_logo.svg' | relative_url }}">
+                            {%- elsif registry == "carpentries" %}
+                            <img alt="Carpentries" class="img-fluid" src="{{ 'assets/img/carpentries_logo.svg' | relative_url }}">
+                            {%- elsif registry == "github" %}
+                            <i class="fa-brands fa-github text-dark fa-2xl my-3"></i>
+                            {%- elsif registry == "everse" %}
+                            <img alt="EVERSE logo" class="img-fluid" src="{{ 'assets/img/everse_logo_crop.png' | relative_url }}">
```

Comment:
We only need to override the TeSS logo - not the whole more-information-tiles include